Fix languagetool findings (#299)
SAP/cloud-sdk: docs/js/features/odata/overview.mdx

### Type-Safe OData Client

- We provide a type-safe OData client to build and execute OData requests in JavaScript and TypeScript.
- We ship pregenerated OData clients for the SAP S/4HANA OData services.
  They are also referred to as the _Virtual Data Model_ or _VDM_.
  You can find a list of all those services in the [SAP API Business Hub](https://api.sap.com/package/SAPS4HANACloud?section=Artifacts) (check the ODATA box).
- We support both OData v2 and OData v4 protocols in the SAP Cloud SDK for JavaScript.

### Code Generator

> If you need a client for an OData service where we do not ship a pregenerated client use our versatile [code generator](generate-typed-odata-v2-and-v4-client-for-javascript-and-typescript) to convert OData service definition into the type-safe Java client library.

Reviewer: ```suggestion
If you want to use an OData service, that has no pregenerated client, you can use the [client generator](generate-typed-odata-v2-and-v4-client-for-javascript-and-typescript) to generate your own client.
```
